### ISSUE TITLE
cache url / timestamp

### ISSUE THREAD

**Issue by BurakKartal1** [ampproject/amp-toolbox#84]
timestamp parameter in url is undefined.

&amp_ts=undefined

```
  const privateKey = req.body.privatekey;
  const url = req.body.url;

  // Generate update-cache URLs.
  const updateCacheUrlProvider = UpdateCacheUrlProvider.create(privateKey);
  updateCacheUrlProvider.calculateFromOriginUrl(url)
    .then(refreshUrlInfos => {
      // Append CORS headers.
      const requestingOrigin = req.headers.origin;
      const requestingSourceOrigin = req.query.__amp_source_origin;
      res.header('Access-Control-Allow-Origin', requestingOrigin);
      res.header('Access-Control-Allow-Credentials', 'true');
      res.setHeader('Access-Control-Expose-Headers', 'AMP-Access-Control-Allow-Source-Origin');
      res.setHeader('AMP-Access-Control-Allow-Source-Origin', requestingSourceOrigin);

    refreshUrlInfos.forEach(cacheUpdateUrlInfo => {
        console.log('Cache ID:' + cacheUpdateUrlInfo.cacheId);
        console.log('Cache Name:' + cacheUpdateUrlInfo.cacheName);
        console.log('cache-update URL:' + cacheUpdateUrlInfo.updateCacheUrl);
      });

      // Sends generated URLs on the response.
      res.send(JSON.stringify({items: refreshUrlInfos}));
    });

```

**Comment by andreban** [comment]
Can you confirm which version of `amp-toolbox-update-cache` is being used? This should have been fixed on #53 (amp-toolbox-update-cache v1.1)

**Comment by andreban** [comment]
@BurakKartal1 I'll close the issue for now. Please, re-open it if you can verify the error on the [latest version](https://www.npmjs.com/package/amp-toolbox-update-cache) of amp-toolbox-update-cache.
